### PR TITLE
Prepare for 2.23.0 release

### PR DESCRIPTION
**Pull request by danyeaw**
This PR bumps the version and prepares for the next release.

### PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I have read, and I am following the [Contributor guide](https://github.com/gaphor/gaphor/blob/main/CONTRIBUTING.md)
- [X] I have read, and I understand the GNOME [Code of Conduct](https://wiki.gnome.org/Foundation/CodeOfConduct)

### PR Type
What kind of change does this PR introduce?

- [ ] Bug fix
- [ ] Feature
- [X] Chore (refactoring, formatting, local variables, other cleanup)
- [ ] Documentation content changes

### What is the current behavior?

Issue Number: N/A

### Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by danyeaw** [comment]
I got distracted and forgot to add all the languages translated on the last changelog entry, let me clean that up before we merge this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+- macOS: Update app icon
+- macOS: Fix ⓘ icon in PropertyPages isn't displayed
+- macOS and Windows: Apply custom window shadow
+- Be very defensive about settings schema sources
+- Fix connecting lines when model is loaded and documentation updates
+- Fix macOS build and documentation generation
+- Update GitHub Actions labeler action to v5
+- Fix pipx install error in Linux
+- Update to Gaphas 4.0
+- Update Russian, Chinese, and German translations
```
Comment (amolenaar) [inline CHANGELOG.md:22]:
```suggestion
- Support types for parameters
- Restore windows in maximized and full-screen state
- Very long element names are now wrapped
- Format files accessed from Flatpak via portals
- Update Property Editor to use Gtk.ColumnView and ListView
- Replace deprecated Gtk.FileChooser by FileDialog
- Replace in-app notifications by Adwaita Toasts
- macOS: Update app icon
- macOS: Fix ⓘ icon in Property Editor isn't displayed
- macOS and Windows: Apply custom window shadow
- Fix Gaphor should work if GSettings schema is not available
- Fix values showing under part compartment in SysML Block elements
- Fix merge-node icon
- Fix connecting lines when model is loaded and documentation updates
- Fix macOS build and documentation generation
- Update Russian, Chinese, and German translations
```

I removed the items that had to do with "internal" stuff (build pipelines, version upgrades).

We can also drop `--filesystem=host` mode in our flatpak ;)

**Review by amolenaar — APPROVED** [review]
Looks good. I suggest some changes to the changelog. (feel free to dismiss)

**Comment by danyeaw** [comment]
Thanks @amolenaar!

**Pull request merged**